tests/devices_test/device_packages_test.py

```diff
@@ -18,9 +18,12 @@ def testPackages(self):
         dev = MDRaidArrayDevice("dev", level="raid1", parents=[dev1,dev2])
         luks = LUKSDevice("luks", parents=[dev])
         packages = luks.packages
+        packages.sort() # sort due to blivet/issues/154
 
         # no duplicates in list of packages
-        self.assertListEqual(packages, list(set(packages)))
+        l = list(set(packages))
+        l.sort() # sort due to blivet/issues/154
+        self.assertListEqual(packages, l)
```

Comment:
Why don't we just compare the sets? The order of the packages doesn't matter to us, AFAICT.